app/models/question.rb

```diff
@@ -0,0 +1,10 @@
+class Question < ActiveRecord::Base
+#has_many :indicators through: :indicator_questions
+
+  validates_presence_of :question
+
+  # QUESTION: Should we allow some way to order the questions?:
+  scope :active, -> { where("active = ?", true) }
+  scope :inactive, -> { where("active = ?",false) }
```

Comment:
Im still a proponent of adding an alphabetical scope for now, but let me know what you guys think. 
Also we should add like a, by_competency scope to get all the questions by competency when we create the indicator_questions model.